src/typing/fields.ml

```diff
@@ -71,6 +71,9 @@ let field_type ctx c pl f p =
 		let monos = Monomorph.spawn_constrained_monos (if pl = [] then (fun t -> t) else apply_params c.cl_params pl) f.cf_params in
 		apply_params l monos f.cf_type
 
+let no_abstract_constructor c p =
```

Comment:
Shouldn't this go to error.ml?

Reply:
No, this is a check the typer makes. The Error module should not have to know what a class of a flag is.